registry/poke_registry.go

```diff
@@ -0,0 +1,26 @@
+package registry
```

Comment:
This package seems a bit of an overkill. It's ok to create all of the required services and dependencies on the main function instead

Reply:
I removed the overkill package.